app/services/data_migrations/backfill_withdrawal_reasons.rb

```diff
@@ -0,0 +1,48 @@
+module DataMigrations
+  class BackfillWithdrawalReasons
+    TIMESTAMP = 20241023160952
+    MANUAL_RUN = false
```

Comment:
If it's only 36k records and with upsert_all I think it's safe to run it automatically but don't really mind if it's manual.

Running it manually allows you to test it in QA and Sandbox first before running it in production.

Reply:
Good shout, I'll change it to true.